ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -0,0 +1,1500 @@
+              description: Time-to-live in range of hops
+              convert_types:
+                - str
+            next_hop_self:
+              type: bool
+            link_bandwidth:
+              type: dict
+              keys:
+                enabled:
+                  type: bool
+                default:
+                  type: str
+                  description: nn.nn(K|M|G) link speed in bits/second
+            rib_in_pre_policy_retain:
+              type: dict
+              keys:
+                enabled:
+                  type: bool
+                all:
+                  type: bool
+            remove_private_as:
+              type: dict
+              description: Remove private AS numbers in outbound AS path
+              keys:
+                enabled:
+                  type: bool
+                all:
+                  type: bool
+                replace_as:
+                  type: bool
+            remove_private_as_ingress:
+              type: dict
+              keys:
+                enabled:
+                  type: bool
+                replace_as:
+                  type: bool
+      neighbor_interfaces:
+        type: list
+        primary_key: name
+        convert_types:
+          - dict
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Interface name
+            remote_as:
+              type: str
+              convert_types:
+                - int
+            peer_group:
+              type: str
+              default: Peer-group name
+            description:
+              type: str
+            peer_filter:
+              type: str
+              description: Peer-filter name
+      aggregate_addresses:
+        type: list
+        primary_key: prefix
+        convert_types:
+          - dict
+        items:
+          type: dict
+          keys:
+            prefix:
+              type: str
+              description: IPv4 prefix "A.B.C.D/E" or IPv6 prefix "A:B:C:D:E:F:G:H/I"
+            advertise_only:
+              type: bool
+            as_set:
+              type: bool
+            summary_only:
+              type: bool
+            attribute_map:
+              type: str
+              description: Route-map name
+            match_map:
+              type: str
+              description: Route-map name
+      redistribute_routes:
+        type: list
+        primary_key: source_protocol
+        convert_types:
+          - dict
+          - list
+        items:
+          type: dict
+          keys:
+            source_protocol:
+              type: str
+            route_map:
+              type: str
+      vlan_aware_bundles:
+        type: list
+        primary_key: name
+        convert_types:
+          - dict
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: VLAN aware bundle name
+            tenant:
```

Comment:
Are you sure about description and tenant here? I can't find anything to indicate we support those keys under the vlan_aware_bundles data model.